docs/dev.rst

```diff
@@ -57,37 +57,20 @@ Managing dependencies
 Python dependencies
 -------------------
 
-Python dependencies for all parts of Socorro are split between two files:
+Python dependencies are maintained in the ``requirements.in`` file and
+"compiled" with dependencies of dependencies in the ``requirements.txt`` file.
 
-1. ``requirements/default.txt``, containing dependencies that Socorro uses
-   directly.
-2. ``requirements/constraints.txt``, containing dependencies required by the
-   dependencies in ``default.txt`` that Socorro does not use directly.
+To add a new dependency, add it to the file and then do::
 
-Dependencies in both files must be pinned and hashed. Use
-`hashin <https://pypi.python.org/pypi/hashin>`_.
-
-For example, to add ``foobar`` version 5::
-
-  $ hashin -r requirements/default.txt foobar==5
-
-If ``foobar`` has any dependencies that would also be installed, you must add
-them to the constraints file::
-
-  $ hashin -r requirements/constraints.txt bazzbiff==4.0
+   $ make shell
+   $ pip-compile --generate-hashes requirements.in
```

Comment:
to be consistent, gotta remove `requirements.in` in both places.
```suggestion
   $ pip-compile --generate-hashes
```

Reply:
:+1: